Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "fastlane"
+gem "fastlane", "2.201.0"
```

Comment:
Note: `fastlane ci` fails if the minimum version of `fastlane` compatible with`xcodebuild_formatter` is not set.

Reply:
Can't we use `>= 2.201.0` or `~> 2.201` rule? Does it have to be exactly 2.201.0?

Reply:
@imaginaris I tested it with `2.201.0` and it works with this exact version. Is it needed to be `~> 2.201`?

Reply:
Ok maybe that's safer approach. Fastlane is known to introduce major bugs or breaking changes in minor versions